cypress/support/commands.ts

```diff
@@ -143,6 +143,7 @@ Cypress.Commands.add("loginByXstate", (username, password = Cypress.env("default
   });
 
   cy.window({ log: false }).then((win) => win.authService.send("LOGIN", { username, password }));
+  cy.location("pathname").should("equal", "/");
```

Comment:
@robertguss Add `return` to the start of this line and append the `then()` block on line 148 to it, so it should look like this:

```
return cy.location("pathname")
.should("equal", "/")
.then((loginUser) => {
    log.set({
      consoleProps() {
        return {
          username,
          password,
          // @ts-ignore
          userId: loginUser.response.body.user.id,
        };
      },
    });
    log.snapshot("after");
    log.end();
  });
});
```

Reply:
@robertguss Just added another assertion inside the `then()`: `cy.getBySel("list-skeleton").should("not.exist");`

Reply:
@robertguss An iteration on the original and edit would be this where I move the assertion inside of the `then` to the main return:

```
return  cy.getBySel("list-skeleton")
.should("not.exist")
.then((loginUser) => {
    log.set({
      consoleProps() {
        return {
          username,
          password,
          // @ts-ignore
          userId: loginUser.response.body.user.id,
        };
      },
    });
    log.snapshot("after");
    log.end();
  });
});


```

Reply:
I tried this originally:
```js
return cy.location("pathname")
.should("equal", "/")
.then((loginUser) => {
```

but it didn't seem to work. I will try again with the other assertion and see what happens.